codalab/lib/bundle_cli.py

```diff
@@ -562,6 +562,9 @@ def resolve_target(
         """
         instance, worksheet_spec, bundle_spec, subpath = parse_target_spec(target_spec)
 
+        if bundle_spec is None:
+            raise UsageError('bundle spec is not missing')
```

Comment:
Don't you mean `Bundle spec is missing`?

Reply:
oh yeah. Thanks for pointing out.